crates/ruff_linter/resources/test/fixtures/flake8_type_checking/quote.py

```diff
@@ -97,3 +97,10 @@ def f():
 
     def func(self) -> DataFrame | list[Series]:
         pass
+
+def f():
+    from django.contrib.auth.models import AbstractBaseUser
+
+    def foo(self, user: AbstractBaseUser["int"], view: "type[CondorBaseViewSet]"):
+        pass
+
```

Comment:
Can you also add a test case for the opposite quote (`AbstractBaseUser['int']`) and one where both are being used (`AbstractBaseUser['int', "str"]`)?